src/components/Select/Select.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import ReactSelect from 'react-select'
+import ReactSelectAsync from 'react-select/async'
+
+import styles from './Select.module.scss'
+
+export const Select = ({ className, isAsync, ...rest }) => {
+  const props = {
+    className: `${className ? className : ''} ${styles.root}`,
+    ...rest,
+  }
+  if (isAsync) {
```

Comment:
i'd love some documentation on when to use isAsync. i don't even know what it does

Reply:
oh is that what you use loadOptions for?

Reply:
At the top of the markdown file I do link to the library:

```md
We leverage [React-Select](https://github.com/JedWatson/react-select) for our single and multi selects. For example, you can add
`menuIsOpen={true}` below as an attribute to `<Select>` and the menu will remain open.
```

I'll add a link in the js file too though.